src/components/Banner/Banner.tsx

```diff
@@ -168,14 +168,14 @@ export const Banner = ({
       />
 
       <div className={clsx(styles['banner__textAndAction'])}>
-        <div className={clsx(styles['banner__textContent'])}>
+        <div>
           {title && (
-            <Heading as={titleAs} size="title-sm" variant="inherit">
+            <Heading as={titleAs} size="title-md" variant={variant}>
```

Comment:
I changed the variant here because I realized we really don't need to be setting the text color in the styles when the text components support that through props. (The color is being defined in the mixin. I'll update the other messaging components before updating the mixin itself.)